rust/src/libnm_dbus/connection/bridge.rs

```diff
@@ -16,51 +16,76 @@
 use std::collections::HashMap;
 use std::convert::TryFrom;
 
-use crate::{dbus_value::value_hash_get_bool, error::NmError};
+use crate::{dbus_value::own_value_to_bool, error::NmError};
 
 #[derive(Debug, Clone, PartialEq, Default)]
 pub struct NmSettingBridge {
     pub stp: Option<bool>,
+    _other: HashMap<String, zvariant::OwnedValue>,
```

Comment:
nit: underscore is really for something that is not used at all but here it is at least populated.

Reply:
The reason I use `_other` instead of `other` because the `other` might be used by NetworkManager as property name someday, I don't want to conflict with that and be inconsistent on all `NmSettingXxx`.

And of course, it is also my old C habit.